internal/workflow_testsuite.go

```diff
@@ -543,6 +544,63 @@ func (e *TestWorkflowEnvironment) OnUpsertMemo(attributes interface{}) *MockCall
 	return e.wrapWorkflowCall(call)
 }
 
+func (e *TestWorkflowEnvironment) OnNexusOperation(
+	service string,
+	operation string,
+	input any,
+	options any,
+) *MockCallWrapper {
+	var call *mock.Call
+	s, ok := e.impl.registry.nexusServices[service]
+	if !ok {
+		panic(fmt.Sprintf(
+			"nexus service %q is not registered with the TestWorkflowEnvironment",
+			service,
+		))
+	}
+	if _, ok := s.GetOperation(operation); !ok {
```

Comment:
I am not sure you should have to pre-register an operation to mock it any more than you have to register an activity to mock it.